vagrant/Vagrantfile

```diff
@@ -0,0 +1,10 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  config.ssh.forward_agent = true
+
+  config.vm.provision :shell, path: "bootstrap.sh"
+  config.vm.provision :shell, privileged: false, path: "bootstrap-user.sh"
```

Comment:
Can we create a shared folder to sync files between vagrant and host?

Reply:
On closer investigation found `/vagrant` is shared, please update docs.

Reply:
Added a note to the vagrant readme